homework_22102020/main.js

```diff
@@ -0,0 +1,90 @@
+// Bài 1
+function printTable(n) {
+    if (n < 1 || n > 10) return
+    console.log('In bảng nhân:')
+    for (let i = 1; i <= 10; i++) {
+        console.log(n + ' x ' + i + ' = ' + n * i)
+    }
+}
+
+// Bài 2
+function getEvennumber(n) {
+    if (n < 1 || n > 30) return
+    for (let i = 1; i <= n; i++) {
+        if (i % 2 === 0) console.log(i)
+    }
+}
+
+// Bài 3
+function total(n) {
+    if (n < 1 || n > 30) return
+    let sum = 1
+    for (let i = 1; i <= n; i++) {
+        sum += i
+    }
+    console.log(sum)
+}
+
+// Bài 4
+function factorials(n) {
+    if (n < 1 || n > 30) return
+    let sum = 1
+    for (let i = 1; i <= n; i++) {
+        sum *= i
+    }
+    console.log(sum)
+}
+
+// Bài 5
+function countEvennumber(arr) {
+    let sum = 0
+    for (let i = 0; i <= arr.length; i++) {
+        if (arr[i] % 2 === 0) sum++
+    } return sum
+}
+
+// Bài 6
+function deleteIteration(arr) {
+    let array = []
```

Comment:
```suggestion
    const array = []
```
array `array` không bị gán lại ở bên dưới, do vậy hãy đặt là `const`